### PR TITLE
[FrameworkBundle] [Workflow]: fixed configuration so service & arguments won't throw an exception

### PR DESCRIPTION
**Pull request by laurensvandeput**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    |no
| Deprecations? |no
| Tests pass?   | yes
| Fixed tickets | #20815
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -268,7 +268,7 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                     ->thenInvalid('"type" and "service" cannot be used together.')
                                 ->end()
                                 ->validate()
-                                    ->ifTrue(function ($v) { return isset($v['arguments']) && isset($v['service']); })
+                                    ->ifTrue(function ($v) { return (isset($v['arguments']) && !empty($v['arguments'])) && isset($v['service']); })
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: isset is not needed then: `return !empty($v['arguments']) &&...`

**Comment by fduch** [comment]
This PR duplicates already submitted https://github.com/symfony/symfony/pull/20760

**Comment by stof** [comment]
Can you add a test covering this to ensure we don't get a regression ?

**Comment by HeahDude** [comment]
Should be closed as duplicate.

**Pull request closed without merging**